projects/swift-protobuf/build.sh

```diff
@@ -24,13 +24,6 @@ cp bin/llvm-symbolizer $OUT/
 )
 
 # build project
-mkdir swift-protobuf-fuzz
-cd swift-protobuf-fuzz
-swift package init --type=executable
-cp $SRC/fuzz_binary.swift Sources/swift-protobuf-fuzz/main.swift
-cp $SRC/Package.swift Package.swift
-cp ../swift-protobuf/Tests/SwiftProtobufTests/unittest.pb.swift Sources/swift-protobuf-fuzz/
-cp ../swift-protobuf/Tests/SwiftProtobufTests/unittest_import.pb.swift Sources/swift-protobuf-fuzz/
-cp ../swift-protobuf/Tests/SwiftProtobufTests/unittest_import_public.pb.swift Sources/swift-protobuf-fuzz/
+cd FuzzTesting
 swift build -c debug -Xswiftc -sanitize=address,fuzzer -Xswiftc -parse-as-library -Xswiftc -static-stdlib -Xswiftc -use-ld=/usr/bin/ld --static-swift-stdlib --sanitize=address
```

Comment:
This seems to have `--sanitize` twice, needed?

Are the specific ld and static-stdlib flags needed or are those the defaults on linux these days?

Reply:
`sanitize` is an argument to `swift build` but it does not accept the value `fuzzer` 
So, I guess we are better with both